### ISSUE TITLE
Patches still being submitted to commitfest?

### ISSUE THREAD

**Issue by unibum** [orioledb/orioledb#362]
Hi,
Wondering if these patches are still being submitted to commitfest? From memory, could be wrong here, but I recall that Orioledb patches were at one point? but maybe I didnt look properly in more recent commitfests. I'm talking buffer maps etc and others which give the performance improvements. Would be great to see this in main PG.
Thanks for reading and answering.

**Comment by AntonOfTheWoods** [comment]
This is actually quite confusing. The patches seem to keep getting refused... But apparently not to the "we will never accept" level. The current links for the commitfest patches all point to `rejected` pages. But then you find they have been recycled into a later commitfest, and are currently waiting. In any case, I think the currently situation is that they are going to be accepted into `pg18` at the very earliest, and given criticity of the changes, possibly not until after that. The `orioledb` docs don't seem to explain exactly what is going on, nor why it is taking so long. Given it has been several years now, it would be nice to understand what the issues are, even if just to satisfy our curiosity...

**Comment by x4m** [comment]
AFAIK current plan for TableAM patches is to finalize design review with TableAM API maintainers and then commit patchset again. But I might misunderstood something, but that's my recollection from unconference session about TableAM API.

**Comment by JamesYuCN** [comment]
I see the following message and hopefully the patch will come in time to merge into PG18

With the last commitfest underway:https://commitfest.postgresql.org/52/
the release team has decided that the feature freeze deadline will be April 8, 2025
